client/app/styles/onboarding.styl

```diff
@@ -27,13 +33,24 @@ header
             display: block
 
         svg
-            max-width: 100%
-            height:    (200 / 16)em
+            width:          4.5em
+            height:         4.5em
+            border-radius:  50%
+            padding:        1.5em
+            box-shadow:     0 0 1em 0 #ccc
```

Comment:
Colors are all stored in the `_colors.styl` file. Can we use the `$grey-02` here instead of `#ccc` ?